EIPS/eip-3690.md

```diff
@@ -0,0 +1,266 @@
+---
+eip: 3690
+title: EOF - JUMPDEST Table
+author: Alex Beregszaszi (@axic), Paweł Bylica (@chfast), Andrei Maiboroda (@gumb0)
+discussions-to: https://ethereum-magicians.org/t/evm-object-format-eof/5727
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-06-23
+requires: 3540, 3670
+---
+
+## Abstract
```

Comment:
I know this is going to fall on deaf ears, but I recommend a Simple Summary.
```suggestion
## Simple Summary
Adds a new type of contract that has section for storing jump destinations.

## Abstract
```

Reply:
Thanks.  I think we'll have to consider that as the abstract is growing, but I'd opt that we do that at once for all the relevant EIPs (need more discussions to come up concise descriptions for all).

If it is not a blocker, I'd opt for merging in the current state.